README.md

```diff
@@ -93,17 +93,17 @@ open-api-based-service
    $ ballerina init
 ```
 
-### Genarate the web service from the Swagger / OpenAPI definition
+### Genarate the web service from the OpenAPI definition
 
-Ballerina language is capable of understanding the Swagger / OpenAPI specifications. We can easily generate the web service just by typing the following command structure in the terminal.
+Ballerina language is capable of understanding the OpenAPI specifications. We can easily generate the web service just by typing the following command structure in the terminal.

```

Comment:
This sounds like it's repeating info from a previous section. @praneesha thoughts?